src/zeekygen/ScriptInfo.cc

```diff
@@ -51,11 +51,11 @@ static void add_summary_rows(const ODesc& id_desc, const vector<string>& cmnts,
                              ReStructuredTextTable* table)
 	{
 	vector<string> row;
-	row.push_back(id_desc.Description());
+	row.emplace_back(id_desc.Description());
 
 	if ( cmnts.empty() )
 		{
-		row.push_back("");
+		row.emplace_back();
```

Comment:
Only a nit as I stumbled over that, is `row.emplace_back("")` flagged by clang-tidy? That would make it look less curious.

Reply:
Calling `row.emplace_back("")` prevents the use of the default constructor. I would actually leave it like that, i.e., just calling `emplace_back()`. The idea behind `emplace` is to have the container construct the object in-place with the provided constructors arguments, so passing no arguments is actually quite common, even if it may look a bit curious.